docs/RELEASE.md

```diff
@@ -0,0 +1,16 @@
+# Release Process
+
+## QA
+
+1. Decide what commit should be used for the release and create a draft release
+  - The description should contain a list of new features
+2. Share release and a Crashlytics Beta link to that commit with the QA team
+  - New features and core features (create Safe, send transactions, restore Safe) MUST be tested with the Mainnet version
```

Comment:
should link to qa google doc be linked here?

Reply:
no I would not link any private docs here